Assets/Expansion1/CityBanners/citybannermanager.lua

```diff
@@ -1063,7 +1072,7 @@ function CityBanner:UpdateColor()
   if (self.m_Type == BANNERTYPE_CITY_CENTER) then
     self.m_Instance.CityBannerFill:SetColor( backColor );
     self.m_Instance.CityBannerFillOver:SetColor( frontColor );
-    self.m_Instance.CityBannerFillOut:SetColor( brighterBackColor );
```

Comment:
brighterBackColor was not defined anywhere.  XP2 version used frontColor in this spot, so I matched that.